_haikus/untitled_cloud_haiku.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: untitled cloud haiku
+author: Mila
+---
+
+Save file to the cloud
```

Comment:
Can you add a `<br>` or two spaces — `  ` to the end of these lines so that the haiku is formatted properly?